Bio/Blast/ParseBlastTable.py

```diff
@@ -30,6 +32,8 @@ def __init__(self, in_rec):
 
 
 class BlastTableRec(object):
+    """Store a Blast Table record, list of Blast Table Entries."""
```

Comment:
This is similar, using `Container for ...` seems like a better fit.

Reply:
Amended.